src/components/DateTime/absolute/getFormattedTime.js

```diff
@@ -0,0 +1,12 @@
+import { timeToStringCommon } from '../utils/timeToStringCommon.jsx';
+
+// Absolute time formatting with 'ago'
+export const getFormattedTime = (value, dateTimeFormat = 'medium', locale = undefined) => {
```

Comment:
So  because you have created a `timeFormatDefaults`, you should be using it here as well